test/unit/atomic/CMakeLists.txt

```diff
@@ -20,5 +20,6 @@ target_link_libraries(
     PRIVATE common)
 
 set_target_properties(${_TARGET_NAME} PROPERTIES FOLDER "test/unit")
+target_compile_definitions(${_TARGET_NAME} PRIVATE "-DTEST_UNIT_ATOMIC")
```

Comment:
Should we do this confitionally only when `ALPAKA_ACC_ANY_BT_OMP5_ENABLE` is set? This would also automatically document why this is necessary.

Reply:
Doing this would require this line to change again in https://github.com/alpaka-group/alpaka/pull/1127 , because they need to be excluded there.

I would rather add a line like this to all test, so each test can be identified by the code in accelerator selection code [test/common/include/alpaka/test/acc/TestAccs.hpp](https://github.com/alpaka-group/alpaka/pull/1126/files#diff-c93c0e3f72fa0a6945dfff2600feafd4). Other new backends, like [SYCL](https://github.com/alpaka-group/alpaka/pull/789) may also want to exclude some test at some stages of development. This could be a saparate PR, then.